src/shared/java/org/truffleruby/shared/TruffleRuby.java

```diff
@@ -18,27 +18,22 @@
     public static final String MIME_TYPE = "application/x-ruby";
     public static final String EXTENSION = ".rb";
     public static final String ENGINE_ID = "truffleruby";
-    public static final String LANGUAGE_VERSION = "2.7.2";
+    public static final String LANGUAGE_VERSION = "2.7.2"; // BuildInformationImpl.INSTANCE.getLanguageVersion();
     public static final String LANGUAGE_REVISION = BuildInformationImpl.INSTANCE.getFullRevision();
     public static final String BOOT_SOURCE_NAME = "main_boot_source";
-    public static final String RUBY_COPYRIGHT = "truffleruby - Copyright (c) 2013-2019 Oracle and/or its affiliates";
+    public static final String RUBY_COPYRIGHT = "truffleruby - Copyright (c) 2013-" + BuildInformationImpl.INSTANCE.getCopyrightYear() + " Oracle and/or its affiliates";
     public static final boolean PRE_INITIALIZE_CONTEXTS = System
             .getProperty("polyglot.image-build-time.PreinitializeContexts") != null;
 
     public static String getVersionString(String implementationName) {
-        final String vm;
-        if (ImageInfo.inImageCode()) {
-            vm = implementationName + " Native";
-        } else {
-            vm = implementationName + " JVM";
-        }
-
         return String.format(
-                "%s %s, like ruby %s, %s [%s-%s]",
+                "%s%s %s, like ruby %s, %s %s [%s-%s]",
                 ENGINE_ID,
+                BuildInformationImpl.INSTANCE.getBuildName() == null ? "" : String.format(" (%s)", BuildInformationImpl.INSTANCE.getBuildName()),
```

Comment:
Could extract that in a local variable, say `String buildName = ...`, for readability?

Reply:
Done.